src/app/api/schemas.py

```diff
@@ -23,9 +20,16 @@ def default_ts_created(cls, v):
         return v or datetime.utcnow()
 
 
-class UserInDb(UserOut):
+# Authentication request
+class UserAuth(UserInfo):
+    password: str
+    scopes: Optional[str]
+
+
+# Creation payload
+class UserCreation(UserInfo):
```

Comment:
I changed it because it is not what an entry looks like in the DB (missing id & created_at). So it seemed misleading to me, but if that's only me, happy to keep the former!